Source/Core/TaskProcessor.js

```diff
@@ -139,8 +141,8 @@ define([
         if (defined(TaskProcessor._loaderConfig)) {
             bootstrapMessage.loaderConfig = TaskProcessor._loaderConfig;
         } else if (defined(require.toUrl)) {
-            var baseUrl = new Uri('..').resolve(new Uri(buildModuleUrl('Workers/cesiumWorkerBootstrapper.js'))).toString();
-            bootstrapMessage.loaderConfig.baseUrl = baseUrl;
+            bootstrapMessage.loaderConfig.baseUrl =
+                getAbsoluteURL('..', new Uri(buildModuleUrl('Workers/cesiumWorkerBootstrapper.js')));
```

Comment:
`new Uri(buildModuleUrl('Workers/cesiumWorkerBootstrapper.js'))` can just be `buildModuleUrl('Workers/cesiumWorkerBootstrapper.js')` since the helper function will create a uri out of it.